src/screens/TopicMemberScreen/index.js

```diff
@@ -1,13 +1,6 @@
 import * as React from 'react';
-import {
-  Animated,
-  InteractionManager,
-  Keyboard,
-  ScrollView,
-  StyleSheet,
-  Platform
-} from 'react-native';
-import {useNavigation, useRoute} from '@react-navigation/native';
+import {Animated, ScrollView, StyleSheet, Platform} from 'react-native';
+import {useRoute} from '@react-navigation/native';
 import axios from 'axios';
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `InteractionManager` and `Keyboard` imports have been removed, and the `useNavigation` import has been removed as well. If these are not used elsewhere in the code, this is a good change for reducing unnecessary imports. However, if they are used elsewhere in the code, this could cause issues.
```diff
- import {
-   Animated,
-   InteractionManager,
-   Keyboard,
-   ScrollView,
-   StyleSheet,
-   Platform
- } from 'react-native';
- import {useNavigation, useRoute} from '@react-navigation/native';
+ import {Animated, ScrollView, StyleSheet, Platform} from 'react-native';
+ import {useRoute} from '@react-navigation/native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->